### ISSUE TITLE
Improve spellcheck

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1190]
It would be good to improve spellcheck and consistency of spellings. For example:

```
JavaScript, instead of javascript / Javascript / js
Node.js, instead of nodejs / NodeJS / node
```

Use caution when doing global find and replaces to not overwrite commands or code blocks that use those strings.

To improve spellcheck, add words to:

- `styles/config/vocabularies/Plone/accept.txt`
- `styles/config/vocabularies/Plone/reject.txt`

Note that when you add a correct spelling in `accept.txt`, then you do not need to add incorrect casings of the same term to `reject.txt`. See https://vale.sh/docs/keys/vocab#case-sensitivity.

## Vale usage

- https://6.docs.plone.org/contributing/documentation/setup-build.html#vale

## Contributing

- https://6.docs.plone.org/contributing/
- https://6.docs.plone.org/contributing/first-time.html
- https://6.docs.plone.org/contributing/documentation/index.html

**Comment by samprasdsouza** [comment]
Hello @stevepiercy , Can I take up this issue ??

**Comment by chirayu-humar** [comment]
i observed that this issue is mentioned in a pull request named https://github.com/plone/plone.restapi-client/pull/5, which is already merged so i want to conferm whether this issue is resolved now or i can start working on this ?

**Comment by stevepiercy** [comment]
That issue is for a different repository and project. Spell checking can always be improved in this repository. Please feel free to work on improving the quality of spelling.

**Comment by chirayu-humar** [comment]
I am working on this issue

**Comment by chirayu-humar** [comment]
i have generated a draft pull request but i am not confident enough that it will work properly because i have not tested it on my local system because i did knew where is this spell check is used further in codebase, to figure it out it will take some time.

**Comment by Akshat2Jain** [comment]
@stevepiercy, plz close this issue as this is already solved

**Comment by stevepiercy** [comment]
@Akshat2Jain no, there is a LOT more work to do. There are still [hundreds of errors, warnings, and suggestions](https://github.com/plone/documentation/pull/1523#issue-1850843452).

**Comment by IsaacRoy** [comment]
Ok sir can I start resolving this issue ?

**Comment by IsaacRoy** [comment]
I have solved the issue and 3 test case passed . Can you just preview it

**Comment by YashasviniSingh** [comment]
Hi, I’d like to work on this issue. Please let me know if I can take it.

**Comment by stevepiercy** [comment]
@YashasviniSingh please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contribute to Documentation](https://6.docs.plone.org/contributing/documentation/).

**Comment by YashasviniSingh** [comment]
Thank you for the pointers sir, I’ll go through the first-time contributor and documentation guidelines and make sure I follow them.
